.ibm/pipelines/openshift-unauth-tests.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+LOGFILE="pr-${GIT_PR_NUMBER}-openshift-tests-${BUILD_NUMBER}"
+
+source .ibm/pipelines/functions.sh
+
+ibmcloud login --apikey "${API_KEY_QE}"
+ibmcloud target -r eu-de
+ibmcloud oc cluster config -c "${CLUSTER_ID}"
+
+(
+    set -e
+    make install
```

Comment:
For consistency with the other scripts, do we want to also export the `DEVFILE_PROXY` env var here?

Reply:
The proxy running in the cluster, it is not accessible, I had to remove it or it contains an error message